apps/challenges/serializers.py

```diff
@@ -67,7 +67,10 @@ class Meta:
             "created_at",
             "queue",
             "worker_cpu_cores",
-            "worker_memory"
+            "worker_memory",
+            "min_worker_instance",
+            "max_worker_instance",
+            "desired_worker_instance"
```

Comment:
You didn't have to modify the relevant test for this to pass CI?

Reply:
I will modify the test.